benchmarking/src/performance_evaluation.py

```diff
@@ -42,15 +44,16 @@ def __init__(
         num_workers: int,
         user_metadata: Dict[str, Any] = {},
         llm_api: str = "sambastudio",
-        generation_mode: str = "stream",
         timeout: int = 600,
     ):
+        base_uri = os.environ.get("SAMBASTUDIO_BASE_URI")
+        
         self.model_name = model_name
         self.results_dir = results_dir
         self.num_workers = num_workers
         self.user_metadata = user_metadata
         self.llm_api = llm_api
-        self.generation_mode = generation_mode
+        self.generation_mode = "stream" if "stream" in base_uri else ""
```

Comment:
same as comment above, is this expected to be in the base uri?

Reply:
this is now a boolean value internally and as explained in previous comment it's being added in URI inside the code as other kits